### PR TITLE
nixpkgs: Bump Version

### PR DESCRIPTION
**Pull request by GTrunSec**
Update the nixpkgs which can be solved version of dependencies and sync version with repo.
- Test nix-build
```
./result/bin/fermatslastmargin                                              
Setting phasers to stun... (port 3000) (ctrl-c to quit)
```
- Test nix-shell
```
nix-shell shell.nix --command "cabal run fermatslastmargin"                 
Up to date
Setting phasers to stun... (port 3000) (ctrl-c to quit)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Distributing PDFs will get you sued for lots of money ( https://en.wikipedia.org
 * git clone this repository
 * run this from cabal `cabal run`
 * point your web browser to `localhost:3000`
-
+* run repository with nix-shell `nix-shell shell.nix --command "cabal run fermatslastmargin"`
```
Comment (chessai) [inline README.md]: the placement of this is confusing. the list here is an ordered list of commands to run. ideally you would actually change the above line, and omit this one.

Comment (chessai) [inline README.md]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Distributing PDFs will get you sued for lots of money ( https://en.wikipedia.org
 * git clone this repository
 * run this from cabal `cabal run`
```
Comment (chessai) [inline README.md]:
```suggestion
* run this from cabal `cabal run` (optionally: `nix-shell --run "cabal run"`)
```

File: README.md (hunk 1 above)
Comment (GTrunSec) [reply README.md]: Thank you for suggesting.

**Comment by shapr** [comment]
Is this all good with you ppls?

**Review by chessai — APPROVED** [review]

**Pull request merged**